frame/alliance/src/types.rs

```diff
@@ -93,3 +93,25 @@ impl Cid {
 		}
 	}
 }
+
+/// Witness data for force_set_members transactions.
+/// Relevant only if executed on an initialized alliance to reset it.
+#[derive(
+	Copy, Clone, Encode, Decode, Eq, PartialEq, RuntimeDebug, MaxEncodedLen, TypeInfo, Default,
+)]
+pub struct ForceSetWitness {
```

Comment:
Perhaps a DQ but I see `Default:default()` being used to set this to `{ 0, 0 }`. Where is the default impl defined?

Reply:
Default trait has standard implementation for most primitives. 
Here we use derive macro to include them for this struct.